gradio/components/audio.py

```diff
@@ -57,7 +58,7 @@ def __init__(
         self,
         value: str | Path | tuple[int, np.ndarray] | Callable | None = None,
         *,
-        source: Literal["upload", "microphone"] | None = None,
+        source: list[Literal["upload", "microphone"]] | None = None,
```

Comment:
Could we rename this to `sources` to match the image component?

Reply:
Let's keep note of this as a deprecation